src/main/java/org/randywebb/wlts/beans/DetailedMember.java

```diff
@@ -20,916 +20,773 @@
+    /**
+    * @param isSpouse
+    *          the isSpouse to set
+    */
+    public void setIsSpouse(Boolean isSpouse) {
+        put("isSpouse", isSpouse);
+    }
+
+    /**
+    * @return the isAdult
+    */
+    public Boolean getIsAdult() {
+        return getBoolean("isAdult");
+    }
+
+    /**
+    * @param isAdult
+    *          the isAdult to set
+    */
+    public void setIsAdult(Boolean isAdult) {
+        put("isAdult", isAdult);
+    }
+
+    /**
+    * @return the fullTimeMissionary
+    */
+    public Boolean getFullTimeMissionary() {
+        return getBoolean("fullTimeMissionary");
+    }
+
+    /**
+    * @param fullTimeMissionary
+    *          the fullTimeMissionary to set
+    */
+    public void setFullTimeMissionary(Boolean fullTimeMissionary) {
+        put("fullTimeMissionary", fullTimeMissionary);
+    }
+
+    /**
+    * @return the formattedMRN
+    */
+    public String getFormattedMRN() {
+        return get("formattedMRN");
+    }
+
+    /**
+    * @param formattedMRN
+    *          the formattedMRN to set
+    */
+    public void setFormattedMRN(String formattedMRN) {
+        put("formattedMRN", formattedMRN);
+    }
+
+    /**
+    * @return the setApart
+    */
+    public Boolean getSetApart() {
+        return getBoolean("setApart");
+    }
+
+    /**
+    * @param setApart
+    *          the setApart to set
+    */
+    public void setSetApart(Boolean setApart) {
+        put("setApart", setApart);
+    }
+
+    /**
+    * @return the formattedBirthDateFull
+    */
+    public String getFormattedBirthDateFull() {
+        return get("formattedBirthDateFull");
+    }
+
+    /**
+    * @param formattedBirthDateFull
+    *          the formattedBirthDateFull to set
+    */
+    public void setFormattedBirthDateFull(String formattedBirthDateFull) {
+        put("formattedBirthDateFull", formattedBirthDateFull);
+    }
+
+    /**
+    * @return the sustainedDate
+    */
+    public Date getSustainedDate() {
+        return getDate("sustainedDate");
+    }
+
+    /**
+    * @param sustainedDate
+    *          the sustainedDate to set
+    */
+    public void setSustainedDate(Date sustainedDate) {
+        put("sustainedDate", sustainedDate);
+    }
+
+    /** Helper function to add "name = value, ".
+        @param <Type> The type of the value to append
+        @param builder the builder to append to
+        @param name The name of the value
+        @param value The value to add
+    */
+    private static <Type> void appendIfNotNull(StringBuilder builder, String name, Type value) {
+        if (value != null) {
+            builder.append(name + "=");
+            builder.append(value);
+            builder.append(", ");
+        }
+    }
+
+    /**
+    *
+    * @see java.lang.Object#toString()
+    */
+    @Override
+    public String toString() {
+        StringBuilder builder = new StringBuilder();
+
+        builder.append("DetailedMember [");
+        appendIfNotNull(builder, "name", getName());
```

Comment:
Ok, this is a nice function call, I like t better than my old logic.
It seems to miss appending a comma between values though, and is thus not equivalent to what I had before. am I missing something?

Reply:
appendIfNotNull appends a trailing comma. Behavior should not change.

Reply:
It does leave a trailing comma. If you want I can clean that up. It will take a little bit of logic change, but pretty minor.

Reply:
Glad to know it leaves a trailing comma. That's much cleaner code!
The trailing comma for getFormattedBirthDateFull() is OK as it is consistent with prior behavior.

Reply:
There was a change in behavior for the last item. I've added logic to prefix with a comma unless it is the first non-null element being added.